src/codec_rav1e.c

```diff
@@ -172,6 +172,11 @@ static avifResult rav1eCodecEncodeImage(avifCodec * codec,
                 goto cleanup;
             }
         }
+        if (encoder->keyframeInterval > 0) {
+            if (rav1e_config_parse_int(rav1eConfig, "key_frame_interval", encoder->keyframeInterval) == -1) {
```

Comment:
Nit: add a comment to document that "key_frame_interval" is the maximum interval between two keyframes.

Reply:
Done.